zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Schema.scala

```diff
@@ -113,13 +180,19 @@ trait Measurement {
 case class Schema(typeName: String,
                   dimension: Seq[Field],
                   measurement: Seq[Field],
-                  primaryKey: Seq[String],
-                  timeField: String
+                  primaryKey: Seq[Field],
+                  timeField: Option[TimeField]
```

Comment:
Another thought is that even though CloudBerry is used for temporal/spatial analytics, do you require every dataset is a temporal dataset? In other words, CloudBerry wouldn't work for datasets without temporal fields?